APMrover2/AP_MotorsUGV.cpp

```diff
@@ -843,7 +843,7 @@ float AP_MotorsUGV::get_scaled_throttle(float throttle) const
 
     // calculate scaler
     const float sign = (throttle < 0.0f) ? -1.0f : 1.0f;
-    const float throttle_pct = constrain_float(throttle, -100.0f, 100.0f) / 100.0f;
+    const float throttle_pct = constrain_float(throttle, -100.0f, 100.0f) * 0.01f;
```

Comment:
So unrelated to your patch here, but this variable has no right to call itself a percentage.